<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="1">

  <testsuite name="EmsApp"
    errors="0" failures="1" hostname="" id="" package="EmsApp" skipped="0"
    tests="1" time="8.359" timestamp="Thu, 13 Aug 2015 12:32:37 GMT">
  
    <testcase name="Demo test Ems App" time="8.359" assertions="4">

    

    

      <failure message="Testing if element &lt;body &gt; div.smart-green &gt; table &gt; tbody &gt; tr:nth-child(6) &gt; td:nth-child(5) &gt; a&gt; is present.">Testing if element &lt;body &gt; div.smart-green &gt; table &gt; tbody &gt; tr:nth-child(6) &gt; td:nth-child(5) &gt; a&gt; is present.: Expected &quot;present&quot; but got: &quot;null&quot;
    at Object.&lt;anonymous&gt; (d:\Node_Project\Nightwatch3\node_modules\nightwatch\lib\core\assertion.js:105:19)
    at HttpRequest.&lt;anonymous&gt; (d:\Node_Project\Nightwatch3\node_modules\nightwatch\lib\index.js:311:20)
    at HttpRequest.emit (events.js:110:17)
    at HttpRequest.&lt;anonymous&gt; (d:\Node_Project\Nightwatch3\node_modules\nightwatch\lib\index.js:344:15)
    at HttpRequest.emit (events.js:118:17)
    at IncomingMessage.&lt;anonymous&gt; (d:\Node_Project\Nightwatch3\node_modules\nightwatch\lib\http\request.js:150:16)
    at IncomingMessage.emit (events.js:129:20)
    at _stream_readable.js:908:16
    at process._tickCallback (node.js:355:11)</failure>

    
    </testcase>
  
  
  </testsuite>
</testsuites>
